config/routes.rb

```diff
@@ -29,4 +29,13 @@
       post "/login", to: "accounts#login"
     end
   end
+
+  resources :challenges do
```

Comment:
we don't have any rails controllers for challenges, so we shouldn't add any routes for these yet
```suggestion
  resources :challenges, only: [] do
```